v2/internal/crdmanagement/crds.go

```diff
@@ -347,3 +353,87 @@ func VersionEqual(a apiextensions.CustomResourceDefinition, b apiextensions.Cust
 
 	return aVersion == bVersion
 }
+
+func GetNonReadyCRDs(ctx context.Context, cfg config.Values, crdManager *Manager) (map[string]apiextensions.CustomResourceDefinition, error) {
```

Comment:
Can you put these 3 functions into a separate file like `helpers` or something in this package?

Possibly can rename `crds.go` to `manager.go`, since that's really what's in here mostly?